components/Filters.js

```diff
@@ -49,25 +49,30 @@ export default function Filters(props) {
           </Button>
           <Button
             outline
+            color="primary"
             onClick={() => props.modeChange('all')}
             active={props.modeSelection === 'all'}
-            className="w-25"
+            className="w-25 font-weight-bold"
           >
             All
           </Button>
         </ButtonGroup>
-        <Form className="m-2">
+        <Form className="m-1">
           <FormGroup>
-            <Label for="from">From Date:</Label>
+            <Label for="from" className="font-weight-bold text-uppercase">
```

Comment:
Should be `htmlFor` instead of `for`, here and below.

Reply:
Updated - what's the difference between the two?

Reply:
React uses `htmlFor` because `for` is a reserved word in JavaScript. Likewise with `className` vs. `class`.

Reply:
ohhh nice